pmoves/services/semantic-cache/main.py

```
@@ -0,0 +1,262 @@
+    headers: dict[str, str],
+) -> StreamingResponse:
+    """SSE streaming passthrough (no caching)."""
+
+    async def stream_generator():
+        try:
+            async with http.stream(
+                "POST", url, json=body, headers=headers
+            ) as resp:
+                async for chunk in resp.aiter_bytes():
+                    yield chunk
+        except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)